omero/developers/Java.txt

```diff
@@ -39,36 +39,41 @@ the dependencies in the ivy.xml files (e.g.
 
 Connect to OMERO
 ----------------
+(**Note:** The usage of the :source:`Gateway.java` <components/blitz/src/omero/gateway/Gateway.java> API is currently still experimental)
```

Comment:
Ah! I didn't spot this yesterday but the 2nd ` is in the wrong place, it needs to be after the address in <>:

``:source:`Gateway.java <components/blitz/src/omero/gateway/Gateway.java>``